sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClientBuilderExtensions.cs

```diff
@@ -34,6 +34,10 @@ public static class KeyClientBuilderExtensions
         /// <summary>
         /// Registers a <see cref="KeyClient"/> instance with the provided <paramref name="vaultUri"/>
         /// </summary>
+        /// <typeparam name="TBuilder">Type of builder to extend.</typeparam>
+        /// <param name="builder">The builder to extend.</param>
+        /// <param name="vaultUri">The URI to a specific key in an Azure Key Vault.</param>
```

Comment:
Let's add an example of a Key URI here so people know what they are looking for.